### ISSUE TITLE
Mainnet Stable Release 1.3.1 not broadcasting transactions not gossiping well?

### ISSUE THREAD

**Issue by EmrePiconbello** [MinaProtocol/mina#12037]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/MinaProtocol/mina/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report. Please post those things in GitHub Discussions: https://github.com/MinaProtocol/mina/discussions

### Description

We send our pool transactions which are +1000 tx at a certain point they stop going through. We assume they got dropped more than a network load/snark issue and try to send a single tx which was processed. After that, because everyone was busy we didn't have time to make calculations and send these missing transactions 

After about 20 hours our transactions started to be included in the network. The start of it was our node producing a block with our transactions. Following that other block producers also started to include the rest of the transactions. We are not sure about the issue or even our logs on its own can be helpful for this but sharing to create some data points.  
logs https://drive.google.com/drive/folders/1q6y9HnftpeNRjviXcb-DaSVNci9jxGLe?usp=sharing

### Steps to Reproduce

1. Send thousands of transactions 
2. Pray :)

Jokes asides this happened to us only once we might avoid this happening earlier because we manually send these transactions again after some time frame. 
...

### Expected Result

Transactions are to be included in blocks by any block producer. Not getting stuck for whatever reason

### Actual Result

While sending a lot of transactions specifically while pools are distributing a lot of issues like these happen while the general assumption was they mostly dropped from mempool. In this instance, they started to process by the network after our nodes produce a block with our transactions. 

### How frequently do you see this issue?

Rarely

### What is the impact of this issue on your ability to run a node?

Low

### Status

```shell
Don't have the status from that time it was months ago but there is nothing irregular.
```

### Additional information

The impact of this exact issue is low since we are not sure how common this happens but the issue with sending transactions is a big problem happening every epoch.

This is the block we created and started our transactions getting included https://minaexplorer.com/block/3NKpXUMxs3Wb5b6sWVtfyHXTzf6DcjtesSKDU8Cn7QixTcA7fdYL

This is the single transaction we send manually https://minaexplorer.com/block/3NLhgFNp5FXYi1xX1PEaKeS9uRr99xKt5k1HeXaKJYJEyMdcFHD3

This is where our transactions stoped
https://minaexplorer.com/block/3NKGJapBuxM4d584Uxm53bNXdr6TGR9EFhunZxZVbPjRwn2eD2MA

**Comment by shimkiv** [comment]
Might be related to https://github.com/MinaProtocol/mina/issues/12031 and https://github.com/MinaProtocol/mina/issues/12032 (depending on env. configuration and runtime changes)

**Comment by EmrePiconbello** [comment]
Might have a relation with it but I can assure no IP change happened. All of our nodes are on static IP some of them are in our control since they are from our own infra.

**Comment by AustinBaggio** [comment]
Unable to reproduce from our side. This was potentially fixed by https://github.com/MinaProtocol/mina/issues/12032. Closing for now, please re-open if this issue persists with steps to reproduce.
